### ISSUE TITLE
Broken with firefox 69.0 ?

### ISSUE THREAD

**Issue by iago-lito** [Peuj/duplicate-tabs-closer#21]
Hey there. I use this addon a lot, and it seems that it doesn't work anymore after recent upgrade to firefox 69.0 : my duplicate tabs remain duplicates until I close them manually. Was this expected?

**Comment by Peuj** [comment]
Hi,

First, the broken behavior is not expected :)

I cannot reproduce the problem with Firefox 69.0.
I assume you use the latest version 3.4.1 of the add-on.

Can you give me more details about your issue and your workflow? (what you do exactly, it happens with any URLs, what are the options used, the close mode,..)

Thanks

**Comment by JulianVolodia** [comment]
...and check what make them (URIs) so special?

**Comment by iago-lito** [comment]
Yupe, thank you for support :)
The issue unfortunately seems to affect all urls.
The version showing up in firefox add-on tab is 3.4.1.

Here is something I have done, trying to be minimal:
- disable all extensions but DTC in the add-on tab.
- restart firefox.
- it opens on https://google.com/ by default on my laptop.
- click the `+` button to open new tab, then write `https://google.com/` in the main bar.
- ⇒ a duplicate of Google is open, but the two tabs remain :\

Alternatively:
- restart firefox with DTC as only enabled extension.
- navigate here to this post.
- right click on the above google link, `Open Link in New Tab`
- repeat
- ⇒ two Google similar google tabs are open.. and remain :'(

Is there any log I could provide?

Here is some output as starter:

```
$ uname -a
Linux etak 5.2.13-arch1-1-ARCH #1 SMP PREEMPT Fri Sep 6 17:52:33 UTC 2019 x86_64 GNU/Linux

$ pacman -Qi firefox
Name            : firefox
Version         : 69.0-1
Description     : Standalone web browser from mozilla.org
Architecture    : x86_64
URL             : https://www.mozilla.org/firefox/
Licenses        : MPL  GPL  LGPL
Groups          : None
Provides        : None
Depends On      : gtk3  mozilla-common  libxt  startup-notification  mime-types
                  dbus-glib  ffmpeg  nss  ttf-font  libpulse
Optional Deps   : networkmanager: Location detection via available WiFi networks
                  [installed]
                  libnotify: Notification integration [installed]
                  pulseaudio: Audio support [installed]
                  speech-dispatcher: Text-to-Speech [installed]
                  hunspell-en_US: Spell checking, American English
Required By     : None
Optional For    : None
Conflicts With  : None
Replaces        : None
Installed Size  : 187.95 MiB
Packager        : Jan Alexander Steffens (heftig) <jan.steffens@gmail.com>
Build Date      : Mon 02 Sep 2019 10:32:11 PM CEST
Install Date    : Mon 09 Sep 2019 03:36:50 PM CEST
Install Reason  : Explicitly installed
Install Script  : No
Validated By    : Signature
```

**Comment by Peuj** [comment]
Can you also send the options used in DTC? 

![image](https://user-images.githubusercontent.com/6862131/64864545-a289dc80-d637-11e9-9b16-e315d838f31d.png)

**Comment by iago-lito** [comment]
HA! I didn't know about these options :) They were as you show: `On duplicate tab detected: Do nothing`. Which is very wrong regarding the behaviour I expect X)

Everything went back to normal after I picked `Close tab automatically` instead.

The only thing left to report was that the value of this option somehow changed on its own when upgrading to firefox 69. This closes the issue for me :)

Thank you for support!

**Comment by JulianVolodia** [comment]
@iago-lito have you any upgrade log which tells us which Firefox did you used then? Or which options changed in your honest opinion?

**Comment by Peuj** [comment]
I don't think it's related to an update of Firefox.
Most probably it's related to the bug (with options not saved) I've introduced in 3.4.0 and fixed in 3.4.1.

**Comment by JulianVolodia** [comment]
Oh, I see. Thanks @Peuj for clarification.
